### ISSUE TITLE
Inquiry regarding "Occlusion Augmentation" strategy

### ISSUE THREAD

**Issue by yoqim** [deepcs233/VividFace#7]
I was hoping you could provide some clarification on the "occlusion augmentation" strategy mentioned in the paper.

Specifically, I have two main questions:

1) Implementation Details: Could you elaborate on the specific implementation details of the occlusion augmentation? For example, what types of occlusion patterns (e.g., random rectangles, face parts, specific objects) were used, and how were the occlusion masks generated or applied to the training data?
2) Occlusion Databases: Did you utilize any external occlusion databases or public datasets (like those featuring masks, sunglasses, hats, etc.) to construct the augmented data? If so, could you please name them?

Thanks.
